cmds/exp/tsort/multiset.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2012-2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import "fmt"
+
+func makeMultiset() multiset {
```

Comment:
Probably newMultiset will be more idiomatic and more consistent with newGraph

Reply:
I considered that, but as this function returns a struct directly rather than a pointer (because the struct only contains one field, so making it a pointer sounds redundant to me), I think `make*` is more appropriate than `new*`. WDYT?

Reply:
I don't think it matters what kind of stuff this function should return. If you create things like this it more idiomatic to call it new*.

For example from google style guide:
```go
package producer

type Thinger struct{ ... }
func (t Thinger) Thing() bool { ... }

func NewThinger() Thinger { return Thinger{ ... } }
```

Reply:
Ahh okay, thank you for pointing me towards the Google Style Guide on this! I remember learning that `make*` was better from a colleague, but I can't remember why now nor did I ask them why, so I'm more than happy to use `new*` instead.

I've just force-pushed a change, see the latest commit.